datasets/ollie/README.md

```diff
@@ -0,0 +1,221 @@
+---
+annotations_creators:
+- machine-generated
+language_creators:
+- crowdsourced
+languages:
+- en
+licenses:
+- other-university-of-washington-academic
+multilinguality:
+- monolingual
+size_categories:
+- n>1M
+source_datasets:
+- original
+task_categories:
+- conditional-text-generation
+- text-retrieval
+task_ids:
+- entity-linking-retrieval
+- fact-checking-retrieval
+- other-stuctured-to-text
+---
+
+# Dataset Card for Ollie
+
+## Table of Contents
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:**
+https://knowitall.github.io/ollie/
+- **Repository:**
+https://knowitall.github.io/ollie/
+- **Paper:**
+@inproceedings{ollie-emnlp12,
+  author = {Mausam and Michael Schmitz and Robert Bart and Stephen Soderland and Oren Etzioni},
+  title = {Open Language Learning for Information Extraction},
+  booktitle = {Proceedings of Conference on Empirical Methods in Natural Language Processing and Computational Natural Language Learning (EMNLP-CONLL)},
+  year = {2012}
+}

```

Comment:
```suggestion
- **Homepage:** [Ollie](https://knowitall.github.io/ollie/)
- **Repository:** [Github](https://github.com/knowitall/ollie)
- **Paper:** [Aclweb](https://www.aclweb.org/anthology/D12-1048/)
```
Here we expect an url for the paper.
I also changed the repository url.

Reply:
Cool. I'll make the change.